### PR TITLE
TTAHUB-28/al view of approvers and status

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

We now show the approver and current status of their review on the various AR report pages. This is currently taking an array and will be easy to hook up once we switch over to multiple approvers.

## How to test

**As a Submitter:**
1. Create an AR
2. Edit and AR
3. View AR that Needs Action
4. View AR that has been Approved

**As a Approver:**
1. View an Approved Report
2. View a report in Review

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-28

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

~- [ ] Staging smoke test completed~

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/ActivityReport/Pages/Review/Approver/Approved.js
Comment (jasalisbury) [inline frontend/src/pages/ActivityReport/Pages/Review/Approver/Approved.js:43]: :+1: should make the switch to multi-approvers easy!

File: frontend/src/pages/ActivityReport/Pages/Review/Approver/index.js
Comment (jasalisbury) [inline frontend/src/pages/ActivityReport/Pages/Review/Approver/index.js:30]: why do we need the `useEffect` here? Are `approvingManager` and `status` sometimes null? If we can remove the `useEffect` we can also simplify the approval status state into a simple variable.

File: frontend/src/pages/ActivityReport/Pages/Review/Submitter/index.js
Comment (jasalisbury) [inline frontend/src/pages/ActivityReport/Pages/Review/Submitter/index.js:40]: I can see why we need the `useEffect` here, a draft report doesn't have an approving manager yet (but can specify one). I could go either way keeping the `useEffect` for the approver for consistency vs removing it for the approver.

**Review by jasalisbury — APPROVED** [review]
Looks good! Just have the question about the `useEffect` on the approver page.

File: frontend/src/pages/ActivityReport/Pages/Review/Approver/index.js
Comment (AdamAdHocTeam) [reply frontend/src/pages/ActivityReport/Pages/Review/Approver/index.js:30]: The idea here was that we have the correct list of approving managers when the page loads or if the selected approving manager changes on the page.

If you think that a simple variable will have the same effect I will change. Let me know your thouhts.

Comment (AdamAdHocTeam) [reply frontend/src/pages/ActivityReport/Pages/Review/Approver/index.js:30]: Scratch my last comment you are correct. The approver cannot change the approving managers from this path. :-)

**Pull request merged**
